Makefile

```
@@ -125,23 +128,27 @@ start-integration: web/src/build/static/app.js bin/goalert bin/psql-lite bin/wai
 jest: node_modules 
 	yarn workspace goalert-web run jest $(JEST_ARGS)
 
-test: node_modules jest
+test: node_modules jest ## Run all unit tests
 	go test -short ./...
 
 force-yarn:
 	yarn install --no-progress --silent --frozen-lockfile --check-files
 
-check: force-yarn generate node_modules
-	# go run ./devtools/ordermigrations -check
-	go vet ./...
-	go run github.com/gordonklaus/ineffassign ./...
-	CGO_ENABLED=0 go run honnef.co/go/tools/cmd/staticcheck ./...
+check: check-go check-js check-code  ## Run all lint checks
+	./devtools/ci/tasks/scripts/codecheck.sh
+
+check-js: force-yarn generate node_modules
 	yarn run fmt
 	yarn run lint
 	yarn workspaces run check
-	./devtools/ci/tasks/scripts/codecheck.sh
 
-check-all: check test smoketest cy-wide-prod-run cy-mobile-prod-run
+check-go:
```

Need to ensure code generation is run _before_ validating
```suggestion
check-go: generate
```